org/meta.go

```diff
@@ -10,4 +10,4 @@ package org
 //
 // We need to always use strings for values so that meta-data is easy to convert
 // into other formats, such as protobuf which has strict type requirements.
-type Meta map[string]string
+type Meta map[Key]string
```

Comment:
Just to double check the semantics of this type: Is metadata of orgs always related to an `org.Key` value?

Reply:
That's the objective, yes. Key has a specific format, so the idea is to make the keys of meta-data more consistent, and a bit easier to define as constants in the code. Here is an example usage: https://github.com/invopop/gobl/blob/f49eb172e84a70fadd316bd7acc2b047253c1321/regions/es/es.go#L136

It could have been done with a String, but this feels easier to check... opinions welcome if you think its overdone!

Reply:
In that case I think `Key` is perfectly fine 👍